c2cgeoportal/scripts/c2ctool.py

```diff
@@ -384,10 +384,10 @@ def step1(self):
         else:
             print("")
             print(self.color_bar)
-            print(
-                "Apply the manual migration steps based on what is in the CONST_CHANGELOG.txt file"
-                " (listed in the `changelog.diff` file)."
-            )
+            text = "Apply the manual migration steps based on what is in the CONST_CHANGELOG.txt file"
+                   " (listed in the `changelog.diff` file)."
+            print(colorize(text, RED)
```

Comment:
I think that red isn't the better color, shouldn't we use yellow?
Also it should be modified for each steps.